services/cinema_hall.py

```diff
@@ -0,0 +1,16 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls():
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: str,
+                       hall_rows: int,
+                       hall_seats_in_row: int) -> None:
+    cinema_halls = CinemaHall.objects.create(
+        name=hall_name,
+        rows=hall_rows,
+        seats_in_row=hall_seats_in_row
+    )
+    return cinema_halls
```

Comment:
The function `create_cinema_hall` is defined to return `None`, but it currently returns the created `CinemaHall` object. If the intention is to return the created object, update the function signature to reflect this change. Otherwise, remove the `return cinema_halls` statement to ensure the function returns `None` as specified.